.circleci/config.yml

```diff
@@ -80,45 +80,53 @@ jobs:
           name: push latest image
           command: docker push "$IMAGE_NAME:latest"
 
-only_on_main_branch: &only_on_main_branch
+build_everywhere: &build_everywhere
+  filters:
+    tags:
+      only: /^\d+\.\d+\.\d+$/
```

Comment:
hmm, doesn't this imply that it runs only on tags that satisfy this regex `/^\d+\.\d+\.\d+$/` ?

Reply:
yeah - do you have a good suggestion?